Bottom: f58aa69c3adee5cef59558b360220f1b21d5fdfb
Top:    89a11c07e3528284e7e17a2863787d1d3d5f5fb5
Author: Christopher Voltz <cjunk@voltz.ws>
Date:   2016-04-29 01:43:43 -0500

Update README and convert to English


---

diff --git a/README.md b/README.md
index e3ade3f..0520977 100644
--- a/README.md
+++ b/README.md
@@ -1,7 +1,8 @@
 tablecalc
 =========
 
-dokuwiki plugin nejen pro FO. Časem bude patrně nahrazen.
-
-Fork: https://www.dokuwiki.org/plugin:tablecalc
+Dokuwiki plugin which adds limited spreadsheet functionality to tables. See
+the original documentation page at: https://www.dokuwiki.org/plugin:tablecalc
 
+Forked from https://github.com/pirati-cz/tablecalc. See the CHANGELOG.md for a
+list of differences.
